src/mpid/ch4/netmod/ofi/ofi_pre.h

```diff
@@ -101,9 +101,11 @@ typedef struct {
 } MPIDI_OFI_lmt_msg_t;
 
 typedef struct {
-    MPIDI_OFI_lmt_msg_payload_t *lmt_msg;
     void *unpack_buffer;
     MPI_Aint pack_size;
+    uint16_t src_offset;
```

Comment:
This is `uint64_t` in `MPIDI_OFI_lmt_msg_payload_t`.

Reply:
Ouch, thanks for checking!

Reply:
👍 this version looks good.